opentelemetry/proto/profiles/v1development/eventprofile.proto

```diff
@@ -0,0 +1,230 @@
+  uint64 value = 7;
+
+  // The number of events that were aggregated into this event. A count value of
+  // 0 should be used for individual events to reduce message size, but 1 is
+  // also allowed and considered to be equivalent. A count larger than 1
+  // indicates that this event is an aggregate of multiple events.
+  uint64 count = 8;
+
+  // The the sum the values of the aggregated events. If an event has a count
+  // and a EventType.default_value, the sum field should be 0, and clients
+  // should derive it by multiplying the count with the default value.
+  uint64 sum = 9;
+
+  // TODO: Add more statistics for aggregated events (e.g. min, max,
+  // stddev, etc) or even histogram sketches?
+}
+
+// Clock describes the clock format used by the EventType. Clocks are expected
+// to be monotonic and include the time when a system is suspended.
+message Clock {
+  // The number of clock cycles per second (Hz). To represent a wall clock with
+  // nanosecond resolution, this would be 1_000_000_000. For a TSC clock, the
+  // value will depend on the TSC frequency of the CPU. The value of this field
+  // impacts the interpretation of Event.time and Event.duration.
+  uint64 frequency = 1;
+  // The start time of the clock in nanoseconds since the Unix epoch.
+  // Implementations SHOULD avoid chosing a start time that falls within a leap
+  // second as the corresponding unix timestamps are ambiguous relative to UTC.
+  fixed64 time_unix_nano = 2;
+  // The indices of the attributes associated with this clock.
+  repeated uint32 attribute_indices = 3;
+}
+
+message EventType {
+  // The name of the event type. Semantic conventions define the names of common
+  // profiling events as well as how they map to existing profiling formats such
+  // as JFR and Go Execution Traces.
+  uint32 name_string_index = 1;
+  // The unit of the Event.value and Event.sum field. Follows the format
+  // described by http://unitsofmeasure.org/ucum.html and MUST be specified
+  // using the UCUM case sensitive (“c/s”) variant. The units "By" for bytes and
+  // "s" for seconds as well as the prefixes "n" (nano), "u" (micro), "m"
+  // (milli), "k" (kilo), "M" (mega) and "G" (giga) SHOULD be supported by all
+  // implementations.
+  uint32 unit_string_index = 2;
+  // The index of the Clock associated with this event type. The absence of this
+  // field indiciates that events associated with this event type are expected
+  // to be aggregates containing a Event.count for the number of events that
+  // were aggregated into this event and an Event.value representing the sum of
+  // the values of the aggregated events.
+  optional uint32 clock_index = 3;
+  // The default value for the Event.value field. This is used when the value
+  // field is the same for all events of this type. For example CPU samples may
+  // typically represent the same value of 10ms.
+  uint64 default_value = 4;
```

Comment:
This feels more like weight than "default value" to me.

Reply:
I took the term `value` from pprof's `Sample.value` field. `weight` would also work for me. I don't feel strongly about it.

Reply:
I see. No strong opinion, we can discuss later if anyone becomes more opinionated.